src/App.tsx

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+export const items = getNumbers(1, 42).map((n) => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const onPageChange = (page: number) => {
+    setCurrentPage(page);
+  };
+
+  const handlePerPageChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setPerPage(parseInt(event.target.value, 10));
+    setCurrentPage(1); // Reset to the first page when changing items per page
+  };
+
+  const startIndex = (currentPage - 1) * perPage;
+  const endIndex = startIndex + perPage;
+  const currentItems = items.slice(startIndex, endIndex);
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
+      <Pagination
+        total={items.length}
+        perPage={perPage}
+        currentPage={currentPage}
+        onPageChange={onPageChange}
+      />
+
       <p className="lead" data-cy="info">
         Page 1 (items 1 - 5 of 42)
```

Comment:
this text should changed and render actual info about which page now selected and which items renered.
it is also reason of failed tests.